packages/fluentui/react-northstar/src/utils/doesNodeContainClick.tsx

```diff
@@ -23,7 +24,7 @@ export const doesNodeContainClick = (
 
     if (target.querySelector('[data-suir-click-target=true]')) {
       _.invoke(e.target, 'removeAttribute', 'data-suir-click-target');
-      return node.contains(e.target as HTMLElement);
+      return elementContains(node, e.target as HTMLElement);
```

Comment:
Change there looks a bit scary as `doesNodeContainClick()` is used widely.

Reply:
However, this is the core purpose of this PR. We want to ensure that clicking into a Fabric Layer does not cause Fluent Popups to close, so clicking into a 'virtual' child needs to be treated as clicking 'within' the element.

Reply:
⚠️ This change causes unit test failure in `Portal-test.tsx`:

```jsx
node.contains(e.target as HTMLElement); // true
elementContains(node, e.target as HTMLElement); // false
```